problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -1,10 +1,81 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob  Compute the following dot products.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item   $\mat{9\\4} \cdot \mat{10\\-3}$
+			\item   $\mat{1\\36\\2} \cdot \mat{0\\0\\1}$
+			\item   $\mat{7\\6\\-3} \cdot \left(\mat{5\\11\\-1} + \mat{-2\\-6\\-1}\right)$
+			\item   $\mat{1\\3\\0\\-5\\5} \cdot \mat{1\\2\\2\\1\\2}$
+			\item   $\frac{1}{2}\mat{2\\5\\4} \cdot \mat{1\\0\\-1}$
 		\end{enumerate}
+		\prob Compute the length of the following vectors.
+		\begin{enumerate}
+			\item $\mat{2\\0}$
+			\item $\mat{1\\2\\3}$
+			\item $4\mat{5\\-6\\15\\2}$
+			\item $\mat{-\frac{\sqrt{10}}{10}\\-\frac{9\sqrt{10}}{50}\\\frac{12\sqrt{10}}{50}}$
+			\item $\mat{\frac{1}{3}\\\frac{\sqrt{3}}{3}\\0\\-\frac{1}{3}\\\frac{2}{3}}$
+		\end{enumerate}
+		\prob Find the angle between the following vectors.
+		\begin{enumerate}
+			\item $\mat{1\\0}$ and $\mat{-3\\4}$
+			\item $\mat{1\\0\\1}$ and $\mat{-5\\4\\-3}$
+			\item $\mat{1\\2\\3}$ and $\mat{-1\\-1\\2}$
+			\item $\mat{2\\4\\6}$ and $\mat{\frac{3}{2}\\3\\\frac{9}{2}}$
+			\item $\mat{0\\1\\0\\1}$ and $\mat{-1\\1\\3\\0}$
+		\end{enumerate}
+		\prob Determine which of the following vectors are orthogonal to eachother.
```

Comment:
Phrasing is weird. There are 11 vectors listed in the problem...This can be made into a more challenging question about creating orthogonal vectors. How about something like, "for each vector, find two \emph{unit} vectors orthogonal to it"